docs/DeveloperGuide.adoc

```diff
@@ -294,36 +295,55 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+| Priority | As a... | I want to... | So that I can...
+| `* * *` | familiar with command line | type commands into a command line interface | perform tasks faster
```

Comment:
Perhaps you could modify the "I want to ..." to make it clearer how it links to the benefit?
Also, `familiar with command line` is not a type of actor.